.pre-commit-config.yaml

```diff
@@ -29,3 +29,7 @@ repos:
       rev: 3.9.2
       hooks:
           - id: flake8
+    - repo: https://github.com/econchick/interrogate
+      rev: 1.5.0
+      hooks:
+          - id: interrogate
```

Comment:
I also think we need to add this into the Makefile (`check_lint`?) to make it run inside github acations.